packages/mobile/src/services/track-download.ts

```diff
@@ -51,19 +51,19 @@ const downloadOne = async ({
   fileUrl: string
   filename: string
   directory: string
-  getFetchConfig: (filePath: string) => RNFetchBlobConfig
+  getFetchConfig: (filePath: string) => ReactNativeBlobUtilConfig
   onFetchComplete?: (path: string) => Promise<void>
 }) => {
   const filePath = directory + '/' + filename
 
   try {
-    const fetchTask = RNFetchBlob.config(getFetchConfig(filePath)).fetch(
+    const fetchTask = ReactNativeBlobUtil.config(getFetchConfig(filePath)).fetch(
       'GET',
       fileUrl
     )
     fetchTasks = [fetchTask]
 
-    // TODO: The RNFetchBlob library is currently broken for download progress events on both platforms.
+    // TODO: The ReactNativeBlobUtil library is currently broken for download progress events on both platforms.
```

Comment:
wonder if this is still true?

Reply:
I'm not sure! Would be good to check after this gets merged, but the UI dropped %age support anyway. Maybe we can bring it back!

Reply:
see other convo